docs/slicer-plot.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a175931f-a0fd-48f7-83cc-9db14d454535",
+   "metadata": {},
+   "source": [
+    "# Slicer plot\n",
```

Comment:
As shown by a user comment on Slack, the name "Slicer" may be misleading?

Reply:
Any better ideas?

Reply:
Ask the users?

Reply:
Greg says `slicer` is fine in the end.